routes/utils.js

```diff
@@ -430,3 +433,72 @@ function _freeId (objType, id) {
         }
     });
 }
+
+
+
+// generic function for send notif (and email)
+
+function get_mail_config () {
+    var MAIL_CONFIG = {};
+    // todo: more and more ugly init...
+    if (CONFIG[MAILER]) { MAIL_CONFIG = CONFIG[MAILER]; }
+    // todo: find a cleaner way to allow registration if no mail are configured
+    if (!MAIL_CONFIG.origin) { MAIL_CONFIG.origin = 'nomail@nomail.org'; }
```

Comment:
if no original email, means there is no contact for the tool/team. Can't think of any real use case where there is no contact email

Reply:
Yes , no real case, only with some specific test env, I will remove it

Reply:
https://github.com/FanchTheSystem/genouestaccountmanager/commit/b208d9a245ff6d07270eff0f1430c17ebb2e185f